### PR TITLE
Fix typos in signals and custom management commands docs.

### PR DESCRIPTION
**Pull request by hramezani**
Continue to [PR 11483](https://github.com/django/django/pull/11483).

### CODE REVIEW

**Comment by hramezani** [comment]
@felixxm 

> Custom management commands example imports Question as Pollso it should work

Yes, right. but in the `closepoll.py` file `handle` function, the example has:
```
    poll.opened = False
    poll.save()
```
We don't have `open` field in `Question` model.

**Comment by felixxm** [comment]
@hramezani It is explained in this doc:
> The ``poll.opened`` attribute does not exist in the :doc:`tutorial</intro/tutorial02>` and was added to ``polls.models.Question`` for this example.

File: docs/howto/custom-management-commands.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ Applications can register their own actions with ``manage.py``. For example,
 you might want to add a ``manage.py`` action for a Django app that you're
 distributing. In this document, we will be building a custom ``closepoll``
 command for the ``polls`` application from the
-:doc:`tutorial</intro/tutorial01>`.
+:doc:`tutorial</intro/tutorial02>`.
```
Comment (felixxm) [inline docs/howto/custom-management-commands.txt]: I think we can leave here `tutorial01` because it starts `Creating the Polls app`.

**Comment by felixxm** [comment]
Merged in a7038adbd02c916315b16939b835f021c2ee8880.

**Pull request closed without merging**